tests/test_Sequence.cc

```diff
@@ -161,6 +185,8 @@ TEST_CASE("Sequence: erase()", "[Sequence]")
         REQUIRE(not seq.empty());
         REQUIRE(3 == seq.size());
         REQUIRE(Step::type_end == (*erase).get_type());
+        REQUIRE(seq.get_error().has_value());
+        REQUIRE(seq.get_error()->get_index().has_value() == false);
```

Comment:
Yes, I slightly prefer the "== false" variant. Probably because it sounds more like a real sentence:
```cpp
REQUIRE(condition == false); // ... require that condition is false ...
REQUIRE_FALSE(condition); // ... require that the following is false: condition ...
```